src/util/typeguard.ts

```diff
@@ -315,7 +315,7 @@ export function isArrayType(
 ): boolean;
 export function isArrayType(
   type: Type | null,
-  assumeType: boolean = false,
+  assumeType = false,
```

Comment:
I've noticed this change in quite a few places. What was the need for this?

Reply:
There as a linting rule that disallowed explicit typing.
`error @typescript-eslint/no-inferrable-types : Type boolean trivially inferred from a boolean literal, remove type annotation.`
Not sure if we want that rule, but it would not lint without removing the type declaration.